src/signed_data.rs

```diff
@@ -165,16 +165,16 @@ pub(crate) fn verify_signature(
     .map_err(|_| Error::InvalidSignatureForPublicKey)
 }
 
-struct SubjectPublicKeyInfo<'a> {
-    algorithm_id_value: untrusted::Input<'a>,
-    key_value: untrusted::Input<'a>,
+pub(crate) struct SubjectPublicKeyInfo<'a> {
+    pub(crate) algorithm_id_value: untrusted::Input<'a>,
```

Comment:
I made the `algorithm_id_value` field crate-visible because it felt like it could be useful down the road, but this branch doesn't use this field (_there's no algorithm ID available when we unmarshal a PKCS8 private key_).

If preferred, we could leave this field private and only change the visibility of the top-level struct + `key_value`.

Reply:
I think we should stick with the minimum visibility now and widen the visibility as we need it.

Reply:
SGTM. Reverted that visibility change.